api/tests/core/subscription/phone_validation/test_api.py

```diff
@@ -74,7 +74,7 @@ class SendSMSTest:
         "pcapi.notifications.sms.backends.sendinblue.sib_api_v3_sdk.TransactionalSMSApi.send_transac_sms",
     )
     @override_settings(SMS_NOTIFICATION_BACKEND="pcapi.notifications.sms.backends.sendinblue.SendinblueBackend")
-    def test_send_sms_success(self, app):
+    def test_send_sms_success(self, mock, app):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Ça fait quoi `mock` ici ?

Reply:
Il faut le mettre sinon la fixture `app` prend la valeur du mock (dû au patch en décorateur)